solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/EmptyView/EmptyView.swift

```diff
@@ -0,0 +1,90 @@
+// Copyright © 2021 Bending Spoons S.p.A. All rights reserved.
+
+import Foundation
+import UIKit
+
+struct EmptyViewConfiguration {
+  let title: String
+  let subtitle: String
+}
+
+class EmptyView: UIView {
+  private var titleLabel: UILabel = {
+    var title = UILabel()
+    title.font = UIFont.boldSystemFont(ofSize: 18)
+    title.text = "No repositories found"
+    title.textColor = .black
+    title.textAlignment = .center
+    title.numberOfLines = 0
+    title.translatesAutoresizingMaskIntoConstraints = false
+    return title
+  }()
+
+  private var subtitleLabel: UILabel = {
+    var subtitleLabel = UILabel()
+    subtitleLabel.font = UIFont.systemFont(ofSize: 16)
+    subtitleLabel.text = "Search for users to see their public repositories here"
+    subtitleLabel.textColor = .gray
```

Comment:
try to use `UIColor.systemXXX`. The `system` colours automatically support dark mode. 😉